lib/pymedphys/tests/dicom/test_header_tweaks.py

```diff
@@ -81,7 +87,8 @@ def test_adjust_machine_name():
     assert adjusted_ds != original_ds
     assert adjusted_ds == expected_ds
 
-    command = "pymedphys dicom adjust-machine-name".split() + [
+    command = get_pymedphys_dicom_cli() + [
```

Comment:
ok, so on top of get_pymedphys_dicom_cli() showing up in a number of places, you are calling it over and over. 
Which suggests that you should have another function that already knows it needs to get that (suggesting lrucache for get_pymedphys_dicom_cli()), and this other function should build a dicom_cli based command with some arguments (which appear to typically be a list of strings)
so something like:
build_dicom_cli_command( *args) or build_dicom_cli_command( my_args: list)

Reply:
Does this comment address your concern?

https://github.com/pymedphys/pymedphys/pull/1192#discussion_r523739729

Reply:
If an issue is created to identify the need for a command construction function that uses get_pymedphys_dicom_cli() internally, then my concern here is addressed.